### DISCUSSION TITLE
有人玩绝区零吗?

### DISCUSSION THREAD

**Discussion by JiangHuai-H** [share121/inter-knot#808] [其他]
![哲](https://github.com/user-attachments/assets/fa7c49fd-f146-4c9f-b656-85af26248c93)

如果有的话可以加我UID:30285285

**Comment by SDCOM-0415** [comment]
本人uid
12339933

**Reply by JiangHuai-H** [reply]
你好

**Comment by Kk-k11** [comment]
我的绳网uid是
10939930

**Reply by JiangHuai-H** [reply]
你好

**Comment by Fzy85** [comment]
虽然但是，我觉得吧，绝区零加好友没多大意义啊

**Reply by JiangHuai-H** [reply]
确实，认识一下

**Comment by YiMie148** [comment]
治安关：通通拷走

**Comment by aaa00073** [comment]
“绳匠小心，空洞不了直接进入，否则，你会被无穷无尽的侵蚀淹没！我们以为你准备好邦布，你只需……”“很简单，我成尊不就是了？”说罢，绳匠一把抓起空洞，顷刻炼化！

**Comment by langxmo** [comment]
都已经到3开头了吗，好可怕

**Comment by sky27950** [comment]
10068010

**Comment by Mayf1ower** [comment]
16852620
